components/PlantImage.js

```diff
@@ -0,0 +1,74 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import SvgIcon, { icons } from "@/components/SvgIcon";
+import { useRouter } from "next/router";
+
+const PlantImageContainer = styled.div`
+  position: relative;
+`;
+
+const StyledImage = styled(Image)`
+  border-radius: 0.5rem;
+  position: relative;
+  ${(props) =>
+    props.$location !== "/[id]" &&
+    `&:hover {
+    cursor: pointer;
+  }`}
+`;
+
+const StyledFavoriteButton = styled.button`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
this should not be necessary since you turn your button into a flex container which only contains one element

Reply:
justify-content is gone